mlos_bench/mlos_bench/tests/__init__.py

```diff
@@ -12,6 +12,11 @@
 from mlos_bench.util import get_class_from_name
 
 
+# A common seed to use to avoid tracking down race conditions and intermingling
+# issues of seeds across tests that run in non-deterministic parallel orders.
+SEED = 42
```

Comment:
In a real running version of mlos_bench, I don't think we'd ever have more than one seed running at a time, so while this may mask places where we might not be setting the seed correctly everywhere, on the other hand I think it's OK to set it to the same value as long as it happens towards the top.
Can consider to do this inside launch.py or run.py in the future as well if we want.